akka-actor/src/main/scala/akka/pattern/BackoffOptions.scala

```diff
@@ -503,7 +503,7 @@ object Backoff {
     childName:    String,
     minBackoff:   java.time.Duration,
     maxBackoff:   java.time.Duration,
-    randomFactor: Double): BackoffOptions =
```

Comment:
This method was marked with `@Deprecated`. Could you add a proper Scala `@deprecated` annotation instead as we have in other places. Might be more in this file.